ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -1320,12 +1326,17 @@ public Object updateState(
     int textBreakStrategy =
         TextAttributeProps.getTextBreakStrategy(paragraphAttributes.getString("textBreakStrategy"));
 
+    @Nullable
+    String errorMessage =
+        props.hasKey("errorMessageAndroid") ? props.getString("errorMessageAndroid") : null;
+
     return ReactTextUpdate.buildReactTextUpdateFromState(
         spanned,
         state.getInt("mostRecentEventCount"),
         TextAttributeProps.getTextAlignment(props, TextLayoutManager.isRTL(attributedString)),
         textBreakStrategy,
         TextAttributeProps.getJustificationMode(props),
-        containsMultipleFragments);
+        containsMultipleFragments,
+        errorMessage);
```

Comment:
This code was introduced to fix the issue https://github.com/fabriziobertoglio1987/react-native-notes/issues/12#issuecomment-1071989570. 

<details><summary>video test without this solution</summary>
<p>

<video src="https://user-images.githubusercontent.com/24992535/158931170-38da40a4-8f0e-4a49-bed4-8ed6f84221c4.mp4" width="1000" />

</p>
</details>


<details><summary>video test with this solution</summary>
<p>

<video src="https://user-images.githubusercontent.com/24992535/158931196-21426511-427c-485f-8975-040e18e557c5.mp4" width="1000" />

</p>
</details>

I reviewed again this solution and found the fabric code responsible for triggering the ReactTextUpdate

The text and paragraphAttributes state is passed to Java from C++

https://github.com/facebook/react-native/blob/05b0d29e765be1f60532bd5e5243f193e8828ddb/ReactCommon/react/renderer/components/textinput/androidtextinput/react/renderer/components/androidtextinput/AndroidTextInputState.cpp#L85-L87

The Java ReactAndroid API receives the state update with the method updateState

https://github.com/facebook/react-native/blob/6780e00fcb515af04ff91fba7f28ff7152e3210f/ReactAndroid/src/main/java/com/facebook/react/uimanager/StateWrapper.java#L15-L20

https://github.com/facebook/react-native/blob/6780e00fcb515af04ff91fba7f28ff7152e3210f/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java#L1283-L1285

My current solution is retrieving the errorMessage from the `props` and adding it to the call to `ReactTextUpdate.buildReactTextUpdateFromState` as 8th parameter, but I believe this is not the best solution:
- other props (for ex. `underlineColorAndroid`) include some c++ mappings in the androidtextinput/*.cpp files
- I believe similar c++ configs to `underlineColorAndroid` should be done for errorMessageAndroid 
- `errorMessageAndroid` should trigger a change in state from the above c++ API

More info are included in https://github.com/fabriziobertoglio1987/react-native-notes/issues/12#issuecomment-1078727833 and https://github.com/fabriziobertoglio1987/react-native-notes/issues/12#issuecomment-1078692146

Reply:
commit https://github.com/facebook/react-native/pull/33468/commits/aca0d9989fd4e403aeec6082c1df542836372e8e